zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/SelectTraceIdsFromSpan.java

```diff
@@ -104,8 +104,7 @@
       Input input =
           new AutoValue_SelectTraceIdsFromSpan_Input(
               serviceName,
-              // % for like, bracing with ░ to ensure no accidental substring match
-              "%░" + annotationKey + "░%",
+              annotationKey,
```

Comment:
Is there a reason we wouldn't want the trailing `%`?   I'm guessing that without the trailing `%` it will just do a strict match vs a partial prefix right?

Reply:
yeah this one is the exact match thing which our api uses